frontend/src/checkJwtExpired.js

```diff
@@ -0,0 +1,10 @@
+import { logoutUser } from './logoutUser';
+
+export const checkJwtExpired = () => {
+  const token_duration = localStorage.getItem('token_duration');
+  if (Date.now() > token_duration) {
+    console.log('TOKEN EXPIRED!');
+    logoutUser();
+  } else {
```

Comment:
What happends when user is logged out? 
Pushed to login page? "Session expired"

To do if we got time.